src/Makefile

```diff
@@ -455,7 +472,7 @@ persist-settings: distclean
 	echo SERVER_LDFLAGS=$(SERVER_LDFLAGS) >> .make-settings
 	echo PREV_FINAL_CFLAGS=$(FINAL_CFLAGS) >> .make-settings
 	echo PREV_FINAL_LDFLAGS=$(FINAL_LDFLAGS) >> .make-settings
-	-(cd ../deps && $(MAKE) $(DEPENDENCY_TARGETS))
+	-(cd ../deps && $(MAKE) $(DEPENDENCY_TARGETS) OPT="$(OPT)" STD="$(STD)" WARN="$(WARN)")
```

Comment:
Removing this line fixes the freebsd build. I guess we are now changing all of the flags in deps during build. It's probably the flto flag, but not sure. It seems like only fpconv and linenoise also use there variables.

Reply:
Yeah, I did not realize that OPT, STD etc are standard variables used by many Makefiles. I will remove them and explicitly specify what to use in fast_float. I won't pass anything from main Makefile. This is unnecessary - now that I think about it.